### DISCUSSION TITLE
[Error] "launch path not accessible" when trying to use the banner function

### DISCUSSION THREAD

**Discussion by cpatrick006** [IBM/mac-ibm-notifications#153] [General]
I am getting the "launch path not accessible" when trying to use the banner function. My alerts seem to work perfectly fine.

macOS 12.6 / Intel

Any ideas?

**Comment by SMartorelli** [comment]
Hi :wave:  have you double-checked all the app names set in the AppComponent.swift file? A typo there might cause that error message

**Comment by cpatrick006** [comment]
It was a typo! Thanks.
